dev/io.openliberty.mcp.internal/src/io/openliberty/mcp/internal/sessions/McpSessionId.java

```diff
@@ -0,0 +1,12 @@
+package io.openliberty.mcp.internal.sessions;
+
+public record McpSessionId(String value) {
```

Comment:
I'd have expected the constructor and the `value()` method of this class to be traced. It's possible that it doesn't because the instrumenter doesn't handle records. Either way please mark this class `@Trivial` or `@Sensitive`.